test/specs/validate-schema/validate-schema.spec.js

```diff
@@ -134,7 +140,13 @@ describe("Invalid APIs (Swagger 2.0 schema validation)", () => {
         }
         catch (err) {
           expect(err).to.be.an.instanceOf(SyntaxError);
-          expect(err.message).to.match(/^Swagger schema validation failed.\n(.*)+/);
+          if (test.openapi) {
```

Comment:
Should we make OpenAPI the fallback here and check for `test.swagger` instead?

Reply:
Oh wait this is just a test so eh

Reply:
yeah just a test, i think its fine for now since the majority of these tests are testing swagger defs